src/data/fintype/card.lean

```diff
@@ -184,6 +206,38 @@ begin
   { simp }
 end
 
+@[to_additive]
+lemma fintype.prod_sigma [fintype α] [fintype β] [decidable_eq β] [comm_monoid γ]
```

Comment:
There is no `sigma` in the type of this lemma. `fintype.prod_fiberwise`? BTW, do we have a version with `finset` and `filter`?

Reply:
Re finset version: yes, see the proof :wink:

Reply:
I meant `∏ x in s, g x = ∏ y, ∏ x in s.filter (λ x, f x = y), g x`.

Reply:
Aah, I see. No I don't think we have it. I'll add it after breakfast.

Reply:
Done.